### ISSUE TITLE
Space within buttons

### ISSUE THREAD

**Issue by devdubc** [givanz/VvvebJs#73]
There is no way to add spacing to button text. You have to use the Source Code editor to accomplish this. Maybe adding an additional text input to the button content are would help bypass dom influences.

**Comment by givanz** [comment]
Hi

What do you mean by button spacing? add padding to button element? you can set padding from Style tab > Padding.

**Comment by devdubc** [comment]
> Hi
> 
> What do you mean by button spacing? add padding to button element? you can set padding from Style tab > Padding.

Sorry I should have been more specific in my description. Example (HelloWorld, Hello World) Just try adding a simple space between 2 or more words within a button cannot be preformed via spacebar press because the buttons influence tries to take over and process. I was able to get around this by simply changing the button component to a href component via inputs.js

**Comment by givanz** [comment]
Yes button tag ignores multiple spaces you can use `&ensp;` or `&nbps;` to add more space between words inside `<buton>` tag.
